static/js/controllers/uicontroller.js

```diff
@@ -82,7 +82,8 @@ define(['jquery', 'underscore', 'bigscreen', 'moment', 'sjcl', 'modernizr', 'web
 			"end": "end1",
 			"dial": "ringtone1",
 			"connect": "connect1",
-			"prompt": "question1"
+			"prompt": "question1",
+			"announce": "message1"
```

Comment:
I added to the api. `message1` was not included yet, which when checking for sounds on my side makes it more readable. I can check for `announce` instead of for `message1`.